Apps/Database/src/Delegates/DBDrugLookupDelegate.cs

```diff
@@ -50,7 +50,8 @@ public class DBDrugLookupDelegate : IDrugLookupDelegate
         public List<DrugProduct> GetDrugProductsByDIN(List<string> drugIdentifiers)
         {
             this.logger.LogTrace($"Getting list of drug products from DB... {JsonSerializer.Serialize(drugIdentifiers)}");
-            List<DrugProduct> retVal = this.dbContext.DrugProduct.Where(dp => drugIdentifiers.Contains(dp.DrugIdentificationNumber))
+            List<string> uniqueDrugIdentifers = drugIdentifiers.Distinct().ToList();
```

Comment:
Is it possible to centralize this code for getting distinct list, as it seems that this is called in multiple places?

Reply:
I moved it to the lower layer in the stack so that it is centralized.  We have 3 different methods that take 3 different lists though.